src/lib/tls/tls_policy.cpp

```diff
@@ -568,9 +568,13 @@ void print_vec(std::ostream& o, const char* key, const std::vector<std::string>&
 void print_vec(std::ostream& o, const char* key, const std::vector<Group_Params>& v) {
    o << key << " = ";
    for(size_t i = 0; i != v.size(); ++i) {
-      o << group_param_to_string(v[i]);
-      if(i != v.size() - 1) {
-         o << ' ';
+      auto name = v[i].to_string();
+
+      if(name.has_value()) {
+         o << name.value();
+         if(i != v.size() - 1) {
+            o << ' ';
+         }
```

Comment:
There's way too little pattern matching in C++. 😞

```suggestion
      if(auto name = v[i].to_string()) {
         o << name.value();
         if(i != v.size() - 1) {
            o << ' ';
         }
      }
```